.github/workflows/github-ci.yml

```diff
@@ -267,6 +296,46 @@ jobs:
           path: vlasiator
           if-no-files-found: error
 
+  build_testpackage_debug_flags_ukkoGPU:
```

Comment:
I'd prefer this to be in line with #1100 so ukko_dgx and with appropriate modules.

Also, might be worth building two versions: with these flags, and also with an extra `-DUSE_WARPACCESSORS=1` to ensure the warp accessors code path also remains valid. These could be in the same job, as consequtive steps.

Reply:
So as soon as #1100 is merged I guess?